stl/inc/xstring

```diff
@@ -1270,6 +1270,10 @@ public:
     /* implicit */ constexpr basic_string_view(_In_z_ const const_pointer _Ntcts) noexcept // strengthened
         : _Mydata(_Ntcts), _Mysize(_Traits::length(_Ntcts)) {}
 
+#if _HAS_CXX23
+    constexpr basic_string_view(nullptr_t) = delete;

```

Comment:
The Editors are going to remove the extraneous `constexpr`s from all three of these deleted functions before the working draft goes out (See https://github.com/cplusplus/draft/pull/4706). I think we should remove them here as well. I'll go ahead and do so and move this to Final Review instead of having it sit through another round-trip.
```suggestion
    basic_string_view(nullptr_t) = delete;
```